turbinia/__init__.py

```diff
@@ -17,34 +17,19 @@
 import os
 import sys
 
-from celery import Celery
-
 from turbinia import config
 
-VERSION = '20150916'
+log = logging.getLogger('turbinia')
+
+VERSION = '20170501'
+
+
+class TurbiniaException(Exception):
+  pass
 
 try:
   config.LoadConfig()
 except config.TurbiniaConfigException as e:
   # pylint: disable=logging-format-interpolation
-  logging.fatal('Could not load Turbinia config: {0:s}'.format(str(e)))
+  log.fatal('Could not load Turbinia config: {0:s}'.format(str(e)))
```

Comment:
Same here, use u'' througout the codebase. Easier to catch unicode problems own the read.